app/views/kudos/show.html.erb

```diff
@@ -2,4 +2,7 @@
   <%= ts("Kudos on") %> <%= link_to @kudo.commentable.title, @referrer %>
 </h3>
 
-<%= render "kudos/kudos", :kudos => @kudo.commentable.kudos.with_pseud.order("created_at DESC"), :guest_kudos_count => @kudo.commentable.guest_kudos_count, :commentable => @kudo.commentable %>
+<%= render "kudos/kudos",
+  kudos: @kudo.commentable.kudos.with_user.includes(:user).by_date,
+  guest_kudos_count: @kudo.commentable.guest_kudos_count,
+  commentable: @kudo.commentable %>
```

Comment:
Also not relevant to this PR, but ... is this view file actually used anywhere? The `KudosController` doesn't seem to have a `show` action defined, and it doesn't seem to be included in `config/routes.rb`, either.

Reply:
Looks unused, I'm removing it here and now.